go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k8snetworkplumbingwg/sriov-network-device-plugin
 
-go 1.13
+go 1.17
```

Comment:
Why thre is 1.17 instead of 1.18?

Reply:
there is no 1.18 version in go.mod files yet